docs/usage_and_configuration/the_basics.md

```diff
@@ -200,6 +200,40 @@ project.
 "No". _Black_ is all about sensible defaults. Applying those defaults will have your
 code in compliance with many other _Black_ formatted projects.
 
+### Linked Configuration feature
+
+You can use the `config = ` field to include configuration options from another
+configuration file. Values set in the original configuration file have precedence over
+those in the included file.
+
+Suppose `pyproject.toml` contains:
+
+```{code-block} toml
+---
+lineno-start: 1
+emphasize-lines: 2,3
+---
+[tools.black]
+config = "linked_config.toml"
+color = true
+```
+
+And `linked_config.toml` contains:
+
+```{code-block} toml
+---
+lineno-start: 1
+emphasize-lines: 2
+---
+[tools.black]
```

Comment:
```suggestion
[tool.black]
```
It's `[tool.black]` not `[tools.black]` https://www.python.org/dev/peps/pep-0518/#tool-table